packages/sdk/src/app/HelioSDK.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  ClusterType,
+  CurrencyService,
+  SolExplorerService,
+  TokenConversionService,
+  ConfigService,
+} from "../domain";
+import { HelioApiAdapter, PaylinkSubmitService } from "../infrastructure";
+
+export class HelioSDK {
+  private static instance: HelioSDK;
```

Comment:
as we talked about not sure singleton is the best design pattern for our SDK given that it will limit us to having only one instance throughout the app